addons/snackpacks/CfgVehicles.inc

```diff
@@ -0,0 +1,54 @@
+#define CLASSES_VEHICLES         \
+    "SWST_GummiShark_Item"         \
+    , "SWST_TigerThymes_Item"  \
+    , "SWST_SonicBooms_Item"
```

Comment:
Make sure you include the other items in here that need to be declared, e.g. `SWST_EggCandy_Item`.